### ISSUE TITLE
Loop Label/Button Issue

### ISSUE THREAD

**Issue by jayharland** [mgibbs189/custom-field-suite#409]
I've just noticed for some of the custom loops I've created, the label and buttons "Loop Row" and "Add Row" respectively, aren't retaining the value set. Some of them show the default value while others show the ones I've set. I've attached an image. 

I checked every where I can think of to find the disconnect and have found nothing. It looks like in the wp_postmeta table the fields are saved correctly, so it's only an issue with displaying.

Any help would be appreciated!

![image](https://user-images.githubusercontent.com/10983166/27612246-440b968e-5b5b-11e7-8dc3-5e442f0f8e10.png)

**Comment by chrisvanpatten** [comment]
This is very odd. Can you share a list of current plugins?

**Comment by jayharland** [comment]
No problem, however I've tried disabling every plugin besides CFS with no success.

Better Search Replace
Custom Field Suite
Duplicate Post*
Duplicator
Enhanced Media Library
Google XML Sitemaps
Heartbeat Control
Imsanity
iThemes Security
Post Type Switcher*
TinyMCE Advanced
W3 Total Cache

I've added an * next to the ones I've recently tried using. I needed to duplicate the menu page and change its post type. That is around the time I noticed the issue; it hasn't always been like this.

I suspect it has to do with the duplication plugin and how it copies that data. I've deleted, saved, re-created some of the the troublesome loops and once they're recreated they work as expected minus the data that was there. I also noticed an issue with these duplicated pages and other fields, text inputs showing a 0 for example and not updating.

If that is the case it may be an issue for that plugin.
